### ISSUE TITLE
Documentation(Activity Index): how to implement celeST's Activity Index measure into WF NTP

### ISSUE THREAD

**Issue by jhparkkkk** [42-AI/Elegant-Elegans#29]
### User Story

I want to find out how to implement a measure from celeST(Activity Index) into wf ntp script,
so that wf_ntp also calculates and returns the same measure as celeST as an output.

### Acceptance Criteria

- [x] Understand the definition of  Activity Index in CeleST paper (more specificaly the concerned parameter) 
- [x] Find out what measures in wf_ntp script could be relevant to calculate activity index
- [x] Write a description of how we calculate the Activity Index in WF_NTP in a markdown `docs/features/activity_index.md`.
    - [x] theoritical definition of the activity index
    - [x] description of the different parameters in the activity index
    - [x] transcription in a pratical way of the theoritical definition. 

### Definition of Done

- [x] Define Activity Index from celeST .
- [x] markdown is `docs/features/activity_index.md`.
- [x] linter is passing

**Comment by cnstll** [comment]
From what I gathered, Activity index is a measure of the area covered (number of pixels?) over a period of time (per min) by a worm.

Some links I used during my research:
* [ CeleST website ](https://dcs-lcsr.github.io/CeleST/)
* [CeleST publication paper](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003702)
* [Video representing Activity index measures](http://celest.mbb.rutgers.edu/supplementary_videos/videos_s2/video_s2_celest_extreme_measures_10of10.mov)

**Comment by jhparkkkk** [comment]
[weekly 3 - 2022.06.30]

CeleST source code

Activity Index is obtained by this formula (cf.  CSTCheckResults.m l. 572): 

sweeping_noTimeNorm(ff0) = areaCoveredNorm(ff0) * temporalFreq(ff0*2) * fileDB(currentVideo).frames_per_second * 60

areaCoveredNorm(ff0) = sum(maskAreaCovered(:) > 0) / (averageWormBodyArea / (1+timeWindowForSweptAreas))

**Comment by cnstll** [comment]
After some research in the source code I can see that the Activity Index is defined with the variable `sweeping_noTimeNorm` ([link to line](https://github.com/DCS-LCSR/CeleST/blob/3ef4d20f3bdfbbbf70e423c74a96ed5f9aa62888/CSTCheckResults.m#L620)).  

The calculus of `sweeping_noTimeNorm` is done [there](https://github.com/DCS-LCSR/CeleST/blob/3ef4d20f3bdfbbbf70e423c74a96ed5f9aa62888/CSTCheckResults.m#L608) : it is a product of a worm area (`areaCoveredNorm`) by a time frequency (`temporalFreq`) by the number of frames per min (fps * 60).  

`temporalFreq` seems to be calculated in the function `CSTComputeMeasuresFromBody` whose definition begins [here](https://github.com/DCS-LCSR/CeleST/blob/3ef4d20f3bdfbbbf70e423c74a96ed5f9aa62888/CSTCheckResults.m#L2174).
ATM, I have a hard time understanding the calculus here as I am missing a lot of mathematical background.  

`areaCoveredNorm` is defined [here ](https://github.com/DCS-LCSR/CeleST/blob/3ef4d20f3bdfbbbf70e423c74a96ed5f9aa62888/CSTCheckResults.m#L607) and depends on `maskAreaCovered`defined [here](https://github.com/DCS-LCSR/CeleST/blob/3ef4d20f3bdfbbbf70e423c74a96ed5f9aa62888/CSTCheckResults.m#L565) and `averageWormBodyArea` defined [here](https://github.com/DCS-LCSR/CeleST/blob/3ef4d20f3bdfbbbf70e423c74a96ed5f9aa62888/CSTCheckResults.m#L566) which also depends on `averageWormBodyMask`.

In a nutshell, the calculus of these variables is still unclear.

**Comment by madvid** [comment]
don't forget to rebase
